README.md

```diff
@@ -105,7 +105,7 @@ This tap:
 
 - US Subscription Endpoint: [https://app.pendo.io/api/v1/aggregation](https://app.pendo.io/api/v1/aggregation)
 - EU Subscription Endpoint: [https://app.eu.pendo.io/api/v1/aggregation](https://app.eu.pendo.io/api/v1/aggregation)
-- Primary key fields: `visitor_id`, `account_id`, `server`, `remote_ip`
+- Primary key fields: `feature_id`, `visitor_id`, `account_id`, `server`, `remote_ip`, `user_agent`, `day` or `hour`
```

Comment:
I see the mismatch here - https://github.com/singer-io/tap-pendo/blob/pendo-pk-changes/tap_pendo/streams.py#L800

Reply:
```suggestion
- Primary key fields: `feature_id`, `visitor_id`, `account_id`, `remote_ip`, `user_agent`, `day` or `hour`
```